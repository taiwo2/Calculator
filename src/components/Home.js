/* eslint-disable react/no-unescaped-entities */
import './style.css';
import React from 'react';

function Home() {
  return (
    <div className="home">
      <h1>Welcome to our page!</h1>
      <p>
        This is a actually a great trick you can do with your iPhones
        calculator app. It's also very easy to do.
        First I'll explain what happens when you perform this trick then I'll explain
        how to do it. As I describe what
        happens imagine you're witnessing it being performed on you.
      </p>
    </div>
  );
}

export default Home;
