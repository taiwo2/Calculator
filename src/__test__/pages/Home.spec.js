import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../../components/Home';

test('should correctly render Home page', () => {
  const component = renderer.create(<Home />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
