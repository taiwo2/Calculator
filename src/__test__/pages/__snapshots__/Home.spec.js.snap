// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Home page 1`] = `
<div
  className="home"
>
  <h1>
    Welcome to our page!
  </h1>
  <div>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae dolores illum omnis! Veritatis perspiciatis a saepe fugit rem illo, quod rerum animi similique aliquam asperiores, libero consequuntur. Vero suscipit obcaecati beatae vel dicta atque architecto perspiciatis laboriosam, quaerat soluta voluptas exercitationem odio laborum optio aliquam blanditiis veniam culpa deserunt natus dignissimos excepturi inventore harum vitae. Placeat dolorem dicta totam nemo, sunt iusto culpa tempore cum qui expedita alias aut voluptatem quidem, dignissimos itaque!
    </p>
    <p>
      Lorem ipsum dolor sit amet consectetur adipisicing elit. Repudiandae dolores illum omnis! Veritatis perspiciatis a saepe fugit rem illo, quod rerum animi similique aliquam asperiores, libero consequuntur. Vero suscipit obcaecati beatae vel dicta atque architecto perspiciatis laboriosam, quaerat soluta voluptas exercitationem odio laborum optio aliquam blanditiis veniam culpa deserunt natus dignissimos excepturi inventore harum vitae.
    </p>
  </div>
</div>
`;
